website/docs/migration/premigration-testing.md

```diff
@@ -22,8 +22,25 @@ As an example, let us say that your application performs some complex query and
 You would do the following:
 
 1. Start FerretDB in `diff-normal` mode.
-   This can be achieved by using the `--mode` [flag](../configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable.
-2. Run `mongosh` and insert some documents.
+
+   This can be achieved by setting the `--mode` [flag](../configuration/flags.md) or `FERRETDB_MODE` environment variable to `diff-normal`.
+   By default, FerretDB starts in normal mode (`--mode=normal`/`FERRETDB_MODE=normal`).
+   For more details, see [operation modes](../configuration/operation-modes.md).
```

Comment:
We already mention modes [here](https://github.com/FerretDB/FerretDB/pull/3673/files#diff-c19a68b74f49bd09602838eb05d801da1864cf5e4058d08eb5fcb31b1c81dc4bR12). Should we remove this to avoid duplication?